tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -202,6 +202,15 @@ bool EnergyPlusFixture::compare_err_stream(std::string const &expected_string, b
     return are_equal;
 }
 
+bool EnergyPlusFixture::compare_err_stream_substring(std::string const &search_string, bool reset_stream)
+{
+    auto const stream_str = this->err_stream->str();
+    bool const found = stream_str.find("Normalization divisor entered as zero") != std::string::npos;
```

Comment:
I think you meant search_string instead of harcoding?

Reply:
No I wanted it to search for this hard coded string.


. 

.


.
...





OK, OK, I'll slow down.  Thank you an awful lot, @jmarrec !  Fix incoming soon - tomorrow.

Reply:
Reading only the first bit on my phone in the email notification cracked me up ;)

Reply:
Clang-tidy 15 definitely catches it for me. I wonder why the cppcheck didn't? And that's because we only run cppcheck in `./src` :)